src/configuration/CalibrationConfig.h

```diff
@@ -61,6 +61,19 @@ namespace SlimeVR {
             float G_off[3];
         };
 
+        struct NXPCalibrationConfig {
+            // accelerometer offsets and correction matrix
+            float A_B[3];
+            float A_Ainv[3][3];
+
+            // magnetometer offsets and correction matrix
+            float M_B[3];
+            float M_Ainv[3][3];
+
+            // raw offsets, determined from gyro at rest
+            float G_off[3];
+        };
```

Comment:
You can use the same calibration config as the MPU9250 as the fields are the same.

Reply:
;(